hypha/static_src/src/app/src/containers/MetaTerm/containers/MetaTermForm/components/MetaTermTreeView/index.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import './styles.scss';
+import styles from './styleTreeView';
+import TreeView from '@material-ui/lab/TreeView';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import TreeItem from '@material-ui/lab/TreeItem';
+import { withStyles } from '@material-ui/core/styles';
+
+
+class MetaTermTreeView extends React.PureComponent {
+
+    state = {
+        expanded : []
+    }
+
+    createTreeItemList = (node) => {
```

Comment:
**Nitpick**: What do you think about renaming this to `renderTreeItem`? `createTreeItemList` sounds like the function intends to create a resource.